deploy/jbossas/kit/jboss-wf9/org/elasticsearch/2.1.0/module.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~
+  ~ ModeShape (http://www.modeshape.org)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+-->
+<module xmlns="urn:jboss:module:1.3" name="org.elasticsearch" slot="${version.elasticsearch}">
+    <resources>
+        <resource-root path="elasticsearch-${version.elasticsearch}.jar" />
+	<resource-root path="HdrHistogram-2.1.6.jar" />
+	<resource-root path="commons-cli-1.3.1.jar" />
+	<resource-root path="compress-lzf-1.0.2.jar" />
+	<resource-root path="guava-18.0.jar" />
+	<resource-root path="hppc-0.7.1.jar" />
+	<resource-root path="joda-convert-1.2.jar" />
+	<resource-root path="joda-time-2.8.2.jar" />
+	<resource-root path="jsr166e-1.1.0.jar" />
+	<resource-root path="netty-3.10.5.Final.jar" />
+	<resource-root path="snakeyaml-1.15.jar" />
+	<resource-root path="spatial4j-0.4.1.jar" />	
+	<resource-root path="t-digest-3.0.jar" />
+	<resource-root path="jackson-core-2.6.2.jar" />
+	<resource-root path="jackson-dataformat-cbor-2.6.2.jar" />
+	<resource-root path="jackson-dataformat-yaml-2.6.2.jar" />
+	<resource-root path="jackson-dataformat-smile-2.6.2.jar" />
+    </resources>
+    <dependencies>
+        <module name="javax.api" export="true"/>
+        <module name="sun.jdk" export="true" >
+           <imports>
+               <include path="sun/misc/Unsafe" />
+           </imports>
+        </module>
+	<module name="org.apache.lucene" slot="5.3.1"/>
```

Comment:
these should use the version properties defined in the parent pom, it makes it easier to maintain that way